src/pages/registrationPage/_tests_/RegistrationPage.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import { fireEvent, render, screen, waitFor, } from '@testing-library/react';
```

Comment:
It appears that your registration form involves a password confirmation field. Consider adding a test to ensure that the password confirmation field is present and behaves as expected.

Reply:
Did you add the test ?